MobileClient/Client/Pages/Homepage.razor

```diff
@@ -3,19 +3,19 @@
 @inherits BasePage
 
 <div class="platformSelectorContainer">
-    <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList(Platform.PC)">
+   <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList(Platform.PC)" Style="background-color: var(--secondary);">
```

Comment:
Style bitte in die rounded-circle Klasse oder in eine eigene, wenn die rounded-circle nicht von uns kommt.

Reply:
Geht nicht wegen Komponente und so